lib/src/screens/dashboard/widgets/filter_list_widget.dart

```diff
@@ -0,0 +1,154 @@
+}
+
+class FilterListWidgetState extends State<FilterListWidget> {
+  late bool ascending;
+  late WalletListOrderType? type;
+
+  @override
+  void initState() {
+    super.initState();
+    ascending = widget.initalAscending;
+    type = widget.initalType;
+  }
+
+  void setSelectedOrderType(WalletListOrderType? orderType) {
+    setState(() {
+      type = orderType;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    const sectionDivider = const HorizontalSectionDivider();
+    return PickerWrapperWidget(
+      onClose: () => {
+        Navigator.of(context).pop([type, ascending])
+      },
+      children: [
+        Padding(
+          padding: EdgeInsets.only(left: 24, right: 24, top: 24),
+          child: ClipRRect(
+            borderRadius: BorderRadius.all(Radius.circular(24)),
+            child: Container(
+              color: Theme.of(context).extension<CakeMenuTheme>()!.backgroundColor,
+              child: Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
+                Padding(
+                  padding: EdgeInsets.all(24.0),
+                  child: Text(
+                    S.of(context).order_by,
+                    style: TextStyle(
+                      color:
+                          Theme.of(context).extension<TransactionTradeTheme>()!.detailsTitlesColor,
+                      fontSize: 16,
+                      fontFamily: 'Lato',
+                      decoration: TextDecoration.none,
+                    ),
+                  ),
+                ),
+                if (type != WalletListOrderType.Custom) ...[
+                  sectionDivider,
+                  SettingsChoicesCell(
+                    ChoicesListItem<ListOrderMode>(
+                      title: "",
+                      items: ListOrderMode.all,
+                      selectedItem: ascending ? ListOrderMode.ascending : ListOrderMode.descending,
+                      onItemSelected: (ListOrderMode listOrderMode) {
+                        setState(() {
+                          ascending = listOrderMode == ListOrderMode.ascending;
+                        });
+                      },
+                    ),
+                  ),
+                ],
+                sectionDivider,
+                RadioListTile(
+                  value: WalletListOrderType.CreationDate,
+                  groupValue: type,
+                  title: Text(
+                    WalletListOrderType.CreationDate.toString(),
+                    style: TextStyle(
+                        color: Theme.of(context).extension<CakeTextTheme>()!.titleColor,
+                        fontSize: 16,
+                        fontFamily: 'Lato',
+                        fontWeight: FontWeight.bold,
+                        decoration: TextDecoration.none),
+                  ),
+                  onChanged: setSelectedOrderType,
+                  activeColor: Theme.of(context).dividerColor,
```

Comment:
let's change this for all radio tiles in this picker to be the same color of the selected item in the `SettingsChoicesCell`